src/main/java/com/minecolonies/util/SchematicWrapper.java

```diff
@@ -548,7 +548,7 @@ public boolean findNextBlockToClear()
 
         }
         //Check for air blocks and if blocks below the hut are different from the schematicWorld
-        while((worldBlockAir() || (progressPos.getY() <= getOffset().getY() && doesSchematicBlockEqualWorldBlock())) && count < Configurations.maxBlocksCheckedByBuilder);
+        while((worldBlockAir() ||  doesSchematicBlockEqualWorldBlock()) && count < Configurations.maxBlocksCheckedByBuilder);
```

Comment:
Shouldn't he now compare all the blocks?
Previously he only compared the ones < y now he compares all of them?